src/effects/nyquist/Nyquist.cpp

```diff
@@ -51,35 +51,36 @@ effects from this one class.
 #include <wx/numformatter.h>
 #include <wx/stdpaths.h>
 
-#include "BasicUI.h"
+#include "../../LabelTrack.h"
+#include "../../NoteTrack.h"
+#include "../../ShuttleGetDefinition.h"
+#include "../../prefs/GUIPrefs.h"
+#include "../../prefs/SpectrogramSettings.h"
+#include "../../tracks/playabletrack/wavetrack/ui/WaveChannelView.h"
+#include "../../tracks/playabletrack/wavetrack/ui/WaveChannelViewConstants.h"
+#include "../../widgets/NumericTextCtrl.h"
+#include "../../widgets/valnum.h"
 #include "../EffectEditor.h"
 #include "../EffectManager.h"
+#include "AudacityMessageBox.h"
+#include "BasicUI.h"
 #include "FileNames.h"
-#include "../../LabelTrack.h"
 #include "Languages.h"
-#include "../../NoteTrack.h"
-#include "TimeTrack.h"
-#include "../../prefs/SpectrogramSettings.h"
 #include "PluginManager.h"
+#include "Prefs.h"
+#include "ProgressDialog.h"
 #include "Project.h"
 #include "ProjectRate.h"
 #include "ShuttleAutomation.h"
-#include "../../ShuttleGetDefinition.h"
 #include "ShuttleGui.h"
-#include "TempDirectory.h"
 #include "SyncLock.h"
+#include "TempDirectory.h"
+#include "TimeTrack.h"
+#include "TimeWarper.h"
 #include "ViewInfo.h"
 #include "WaveClip.h"
 #include "WaveTrack.h"
-#include "../../widgets/valnum.h"
-#include "AudacityMessageBox.h"
-#include "Prefs.h"
 #include "wxFileNameWrapper.h"
-#include "../../prefs/GUIPrefs.h"
-#include "../../tracks/playabletrack/wavetrack/ui/WaveChannelView.h"
-#include "../../tracks/playabletrack/wavetrack/ui/WaveChannelViewConstants.h"
-#include "../../widgets/NumericTextCtrl.h"
-#include "ProgressDialog.h"
```

Comment:
Remember, I prefer to keep include directives sorted by the last path component only, because path prefixes will often change (shorten) as we make more libraries.  So omit this please.